### ISSUE TITLE
Antoninus of Piacenza (stoa0029d) is a wrong attribution

### ISSUE THREAD

**Issue by PonteIneptique** [OpenGreekAndLatin/csel-dev#303]
The Itinerarium is in fact a wrong but common attribution apparently ( https://en.wikipedia.org/wiki/Anonymous_pilgrim_of_Piacenza ). I was dating some texts of the corpus when I came accross this one. I don't know where to put an issue for the catalog though.

Given the fact that we do not have text of the real Antoninus apparently, we could just rename this textgroup.

**Comment by sonofmun** [comment]
I am not so sure how we should deal with this. If it is a common misattribution and not simply an error with assigning the URN, I would leave it the way that the edition attributes it (probably to Antoninus) and put something in the extended metadata saying that it is equivalent to the other URN. But I am not exactly sure how to do this. Is there some sort of sameAs element that we could use? How do we do it in the catalog? 
@AlisonBabeu Do you have any input here?

**Comment by PonteIneptique** [comment]
It used to be attributed in th early 19th but it seems to have been given to "pilgrims" since a good time. Latest article stating this was this ( http://www.persee.fr/doc/rebyz_0766-5598_1998_num_56_1_1962_t1_0317_0000_2 )

**Comment by AlisonBabeu** [comment]
Well @sonofmun and @PonteIneptique I'm a bit confused to begin with. The STOA ID I have for Antoninus of Piacenza is stoa0028e (I don't have any author with the ID of stoa0029d), and we do have one text for this "author" here (https://github.com/OpenGreekAndLatin/csel-dev/blob/master/data/stoa0028e/stoa001/stoa0028e.stoa001.opp-lat1.xml).

From looking at the catalog record for this work
I record the information regarding the author of this work using the role of "attributed author"

```xml
<mods:name authority="naf" authorityURI="http://id.loc.gov/authorities/names/" type="personal" valueURI="http://id.loc.gov/authorities/names/no2002112746"><mods:namePart>Antonino,</mods:namePart>
<mods:namePart type="termsOfAddress">of Piacenza, Saint,</mods:namePart>
<mods:namePart type="date">270-</mods:namePart>
<mods:role><mods:roleTerm authority="marcrelator" type="text">
attributed author</mods:roleTerm></mods:role>
<mods:displayForm>Antoninus Placentinus</mods:displayForm>
</mods:name>`
```
One problem with the current catalog display is that it doesn't display any role other than "creator" alas.  Typically if there is a new author attribution or more than one author I will add another separate author statement to the MODS record to indicate the disputed authorship.

```xml
<mods:name authority="none" ><mods:namePart>Anonymous pilgrim of Piacenza</mods:namePart>
<mods:role>
<mods:roleTerm authority="marcrelator" type="text">
creator</mods:roleTerm>
</mods:role>
<mods:displayForm>Antoninus Placentinus</mods:displayForm>
</mods:name>
```
Does this make any sense?

**Comment by PonteIneptique** [comment]
Yes definitely. So, the textgroup being only for this author, I assume it can be renamed to Pilgrims in the __cts__ ?

**Comment by AlisonBabeu** [comment]
I think that would be perfectly appropriate.
